plugins/parsers/xpath/README.md

```diff
@@ -13,7 +13,7 @@ For supported XPath functions check [the underlying XPath library][xpath lib].
 | [Extensible Markup Language (XML)][xml] | `"xml"`               |         |
 | [JSON][json]                            | `"xpath_json"`        |         |
 | [MessagePack][msgpack]                  | `"xpath_msgpack"`     |         |
-| [Protocol buffers][protobuf]            | `"xpath_protobuf"`    | [see additional parameters](protocol-buffers-additiona-settings)|
+| [Protocol buffers][protobuf]            | `"xpath_protobuf"`    | [see additional parameters](protocol-buffers-additional-settings)|
```

Comment:
I think this requires a `#` at the beginning (https://stackoverflow.com/questions/6695439/how-to-link-to-a-named-anchor-in-multimarkdown)

```suggestion
| [Protocol buffers][protobuf]            | `"xpath_protobuf"`    | [see additional parameters](#protocol-buffers-additional-settings)|
```

Reply:
That explains why when i was trying `protocol-buffers-additional-settings` I was still getting a 404 :)